File: iniciante/1014/1014.py
x = int(input())      # Lê um número inteiro representando a distância total percorrida (em km)
y = float(input())    # Lê um número real representando o total de combustível gasto (em litros)

# Calcula o consumo médio dividindo a distância pelo combustível gasto
consumo_medio = x / y

# Imprime o consumo médio com 3 casas decimais, seguido de "km/l"
print(f"{consumo_medio:.3f} km/l")

File: iniciante/1014/1014.c
#include <stdio.h> // Biblioteca padrão de entrada e saída (ex: scanf, printf)

int main()
{
  int x;                  // Declara uma variável inteira para a distância percorrida (em km)
  float y, consumo_medio; // Declara duas variáveis float: y (combustível gasto em litros) e consumo_medio

  scanf("%d", &x); // Lê o valor inteiro da distância
  scanf("%f", &y); // Lê o valor real do combustível gasto

  consumo_medio = x / y; // Calcula o consumo médio (km por litro)

  printf("%.3f km/l\n", consumo_medio); // Exibe o resultado com 3 casas decimais, seguido de "km/l"

  return 0; // Indica que o programa terminou com sucesso
}
